lib/elixir/test/elixir/calendar_test.exs

```diff
@@ -878,4 +887,179 @@ defmodule DateTimeTest do
 
     assert DateTime.diff(dt1, dt2) == 3_281_904_000
   end
+
+  describe "from_naive" do
+    test "uses default time zone database from config" do
+      Calendar.put_time_zone_database(FakeTimeZoneDatabase)
```

Comment:
This test module is has `async` set to `true` for tests. I assume this and `Calendar.put_time_zone_database` can cause race conditions. Should the tests that call `Calendar.put_time_zone_database` be put in a separate test module? Or just make the existing module `async: false`.

cc @josevalim

Reply:
Pushed!